cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -0,0 +1,354 @@
+    # collection after data has been fetched for the last hour.
+
+    appliance.server.settings.disable_server_roles(
+        'ems_metrics_coordinator', 'ems_metrics_collector')
+    rc, out = appliance.ssh_client.run_rails_command(
+        "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
+        vm.perf_rollup_range(4.hour.ago.utc, Time.now.utc,'realtime')\"".
+        format(provider.id, repr(vm_name)))
+    assert rc == 0, "Failed to rollup VM C&U data:".format(out)
+
+    wait_for(verify_records_rollups_table, [appliance, provider], timeout=600, fail_condition=False,
+        message='Waiting for hourly rollups')
+
+    # Since we are collecting C&U data for > 1 hour, there will be multiple hourly records per VM
+    # in the metric_rollups DB table.The values from these hourly records are summed up.
+
+    with appliance.db.client.transaction:
+        result = (
+            appliance.db.client.session.query(rollups.id)
+            .join(ems, rollups.parent_ems_id == ems.id)
+            .filter(rollups.capture_interval_name == 'hourly', rollups.resource_name == vm_name,
+            ems.name == provider.name, rollups.timestamp >= date.today())
+        )
+
+    for record in appliance.db.client.session.query(rollups).filter(
+            rollups.id.in_(result.subquery())):
+        consumed_hours = consumed_hours + 1
+        if (record.cpu_usagemhz_rate_average or
+           record.cpu_usage_rate_average or
+           record.derived_memory_used or
+           record.net_usage_rate_average or
+           record.disk_usage_rate_average):
+            average_cpu_used_in_mhz = average_cpu_used_in_mhz + record.cpu_usagemhz_rate_average
+            average_memory_used_in_mb = average_memory_used_in_mb + record.derived_memory_used
+            average_network_io = average_network_io + record.net_usage_rate_average
+            average_disk_io = average_disk_io + record.disk_usage_rate_average
+
+    for record in appliance.db.client.session.query(rollups).filter(
+            rollups.id.in_(result.subquery())):
+        if record.derived_vm_used_disk_storage:
+            average_storage_used = average_storage_used + record.derived_vm_used_disk_storage
+
+    # Convert storage used in Bytes to GB
+    average_storage_used = average_storage_used * math.pow(2, -30)
+
+    return {"cpu_used": average_cpu_used_in_mhz,
+            "memory_used": average_memory_used_in_mb,
+            "network_io": average_network_io,
+            "disk_io_used": average_disk_io,
+            "storage_used": average_storage_used,
+            "consumed_hours": consumed_hours}
+
+
+@pytest.yield_fixture(scope="module")
+def metering_report(vm_ownership, provider):
+    # Create a Metering report based; Queue the report.
+    owner = vm_ownership
+    data = {
+        'menu_name': 'cb_' + provider.name,
+        'title': 'cb_' + provider.name,
+        'base_report_on': 'Metering for VMs',
```

Comment:
**Required**

This is causing a NoSuchElement exception in PRT, this option isn't found in the bootstrapselect